src/BuiltInTools/dotnet-watch/HotReload/BlazorWebAssemblyDeltaApplier.cs

```diff
@@ -66,6 +67,30 @@ public async ValueTask ReportDiagnosticsAsync(DotNetWatchContext context, IEnume
             }
         }
 
+        private async ValueTask<bool> VerifyDeltaApplied(DotNetWatchContext context, CancellationToken cancellationToken)
+        {
+            var _receiveBuffer = new byte[1];
+            try 
+            {
+                var result = await context.BrowserRefreshServer.ReceiveAsync(_receiveBuffer, cancellationToken);
```

Comment:
I think this needs a timeout - like 5 seconds or so. Otherwise, wouldn't this wait indefinitely in the failure case?

Reply:
Along the same vein, should we return 0 when it wasn't successful?